packages/forms-web-app/src/dynamic-forms/section.js

```diff
@@ -69,7 +70,10 @@ class Section {
 		let answerCount = 0;
 
 		for (let question of this.questions) {
-			const isRequired = question.validators?.some((item) => item instanceof RequiredValidator);
+			//todo: rather than use instanceof can isRequired be a property on BaseValidator?
```

Comment:
So I originally chose the use of instanceof
The reason I suggested this on second viewing is that when creating a new validator there is nothing warning you that this happens in the section class which feels like it will break at some point (i.e. I had to mention it on the ticket)
If we move it to a property on the validator I'm thinking we can enforce that it is explicitly set.

If there's a better way of calculating the section status that avoids needing to check the validators then I guess that could improve things as well.

Reply:
The other way to potentially check is a helper function on the section or journey class that evaluates if the question is on the journey and on the response